challenge_storing_groceries/scripts/rospy

```diff
@@ -0,0 +1,525 @@
+%!PS-Adobe-3.0
```

Comment:
What's this file? Do we need to add it? The filename is quite confusing to me

Reply:
That's a postscript file, common in the era before PDF existed. Basically it's a full programming language to describe what to print onto the paper.
Given its Creator tag (an image processing tool), likely it's an image. After downloading I got a page-filling, fully black rectangle, ie a fully black image.